dataverse-dist/src/main/assembly/moxy-patched-assembly.xml

```diff
@@ -0,0 +1,32 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.1.0 http://maven.apache.org/xsd/assembly-2.1.0.xsd">
+
+    <formats>
+        <format>jar</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+
+    <fileSets>
+        <fileSet>
+            <directory>src/main/moxy-patched</directory>
+            <outputDirectory>/</outputDirectory>
+        </fileSet>
+    </fileSets>
+
+    <dependencySets>
+        <dependencySet>
+            <scope>runtime</scope>
+            <includes>
+                <include>org.eclipse.persistence:org.eclipse.persistence.moxy</include>
```

Comment:
Not sure why - but produced jar contains only MANIFEST.MF for me. I checked if application will run with this jar in modules/ dir in glassfish and it does. Although I think that we should have jar with all the classes

Reply:
I messed up the scope, good catch, fixed now.